be/src/exec/hdfs_scanner_text.cpp

```diff
@@ -308,6 +308,7 @@ Status HdfsTextScanner::parse_csv(int chunk_size, ChunkPtr* chunk) {
 
     csv::Converter::Options options;
     // Use to custom Hive array format
+    options.is_hive = true;
```

Comment:
why do we need this?

Reply:
if you wanna separate the logic of checking allowed max length, you should add a conf call `max_length` instead of add this

Reply:
Because CSV converter is used by broker load and dla. They have different check behavior. We need a flag to distinguish it.